src/app/services/game-play.service.ts

```diff
@@ -14,12 +16,6 @@ interface Game {
   currentGuess: string;
 }
 
```

Comment:
Sorry this is on the wrong line but I just noticed the `game.answer!` inside `answer$` in this class. I think you'll get an error if you subscribe to `answer$` before the game has an answer with the code written that way. Here's another option:
```suggestion
  public readonly answer$: Observable<Answer> = this.game$.pipe(
    map(game => game.answer),
    filter((answer): answer is Answer => !!answer)
  )
```
See this if you're interested in the `answer is Answer` part: https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates